packages/rancher-logging/overlay/templates/outputs/elasticsearch/output.yaml

```diff
@@ -12,32 +12,32 @@ spec:
   {{- if .Values.elasticsearch.user }}
 user: {{ .Values.elasticsearch.user }}
   {{- end}}
-  {{- if .Values.elasticsearch.password }}
+{{- if .Values.elasticsearch.password.secret_name }}
 password:
```

Comment:
What happens if elastic-search is enabled but password isn't passed, it just fails ?

Reply:
es does not require a password so it depends on the setup, if users do not provide a password to a secured cluster they will get errors in their fluentd.